pennylane_forest/qpu.py

```diff
@@ -152,6 +149,10 @@ def __init__(
             if timeout is not None:
                 self.qc.compiler.client.timeout = timeout
 
+        num_wires = len(self.qc.qubits())
+
+        super(QVMDevice, self).__init__(num_wires, shots, **kwargs)
+
```

Comment:
To explain what's going on: before we could infer the qubit number from the lattice. Now the best way seems to be to load the device and query the number of qubits.

Beforehand, we ran `super(QVMDevice, self).__init__` to create a connection and then loaded the qc. However, we need the number of wires as an argument in `super(QVMDevice, self).__init__`.

Now, we create a connection using the `_get_connection` method, then load the qc, then run `init` with the number of wires evaluated.

Reply:
That explains it well. Thanks @trbromley!